src/taskgraph/main.py

```diff
@@ -228,30 +228,32 @@ def logfile(spec):
 
 @command(
     "tasks",
-    help="Show all tasks in the taskgraph.",
+    help="Show the full task set in the taskgraph. The full task set includes all tasks defined by any kind, without edges (dependencies) between them.",
     defaults={"graph_attr": "full_task_set"},
 )
 @command(
-    "full", help="Show the full taskgraph.", defaults={"graph_attr": "full_task_graph"}
+    "full",
+    help="Show the full taskgraph. The full taskgraph consists of the full task set, with edges (dependencies) between tasks.",
```

Comment:
nit: s/taskgraph/task graph

While we're making things less confusing, let's make it clear that we're not talking about the package here. Note this comment applies to all the commands.